src/Formik.tsx

```diff
@@ -765,8 +784,8 @@ export function useFormik<Values extends FormikValues = FormikValues>({
       const field: FieldInputProps<any> = {
         name,
         value: valueState,
-        onChange: handleChange,
-        onBlur: handleBlur,
+        onChange: (e: any) => executeChange(e, name),
```

Comment:
just curry handleChange(name)

Reply:
Yep, that was the first idea, but I'm not good at TS generic types, and I'm getting error type error when I'm using `onChange: handleChange(name)`...

Reply:
Having a hard time understanding what generic type needs to change [here](https://github.com/jaredpalmer/formik/blob/master/src/types.tsx#L132-L134)? I am wanting this feature and was going to finish PR but also not a typescript 10X Engineer.

Reply:
It's simply impossible with generics in current TS version :) (link to the TS issue is correct, we are experiencing exactly the same problem).
IMO we do not need to use `handleChange` here since it's just do the same thing as my code above

IMO 2 possible ways of usage `handleChange` is a kind of "magic" behaviour and it may leads to bugs. I prefer never try to get the name of the field from an `event` and always bind the name to handler.